toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/SceneView.kt

```diff
@@ -211,8 +211,41 @@ public fun SceneView(
         content = content
     )
 }
+
 /**
- * A compose equivalent of the view-based [SceneView].
+ * A composable UI element that displays three-dimensional (3D) geographic content defined by an `ArcGISScene`.
+ *
+ * A composable scene view displays `ArcGISScene` layers and graphics in 3D. It uses a `Camera` to control
+ * the visible area (extent) of the `ArcGISScene` (see [cameraController]) and supports user interactions such as pan,
+ * zoom, tilt, and rotate.
+ * It also provides access to the underlying layer data in the scene.
+ *
+ * User interactions (pan, zoom, tilt, rotate, identify, selection) are supported using touch, mouse, keyboard,
+ * or pen/pencil input on the composable. If required, you can override keys, clicks, and gestures to provide
+ * a specific user experience by using callbacks such as:
+ * - [onPan]
+ * - [onSingleTapConfirmed]
+ * - [onDoubleTap]
+ * - etc.
```

Comment:
This part doesn't fully apply currently. I suggest:
```suggestion
 * User interactions (pan, zoom, tilt, rotate, identify, selection) are supported using touch interaction on the
 * composable. If required, you can respond to certain gesture events to provide
 * a specific user experience by collecting gesture flows such as:
 * - [onPan]
 * - [onSingleTapConfirmed]
 * - [onDoubleTap]
```